mods/persistence/controllers/subsystems/initialization/chargen.dm

```diff
@@ -105,7 +105,6 @@ SUBSYSTEM_DEF(chargen)
 		var/mess = "'[victim]' (CKEY: [victim.ckey]) spawned outside chargen for some reasons."
 		log_warning(mess)
 		message_staff(mess)
-	SSpersistence.AddToLimbo(victim.mind, victim.mind.unique_id, LIMBO_MIND, victim.mind.key, victim.mind.current.real_name, TRUE)
```

Comment:
I think the reason brawler did that was to reserve the character name in the db and save the character, since the character creator doesn't retain the design anymore.

Reply:
The character creator actually does still retain the design, at least it does on my local copy. The check in preferences now additionally checks for the names of player minds, so that should disallow overlaps while the player is still in chargen. Serializing the player while in the chargen pod is a bad idea, since if the server were to crash, they would wake up on Outreach without having gone through the character creation process.